src/Directory.Build.props

```diff
@@ -0,0 +1,30 @@
+<Project>
```

Comment:
I may pull this into CSharpTemplate if you don't beat me to it.

Reply:
I hadn't been thinking about it, but it would make making future repos with multiple projects easier.